libmachine/filestore.go

```diff
@@ -19,7 +19,11 @@ type Filestore struct {
 }
 
 func NewFilestore(rootPath string, caCert string, privateKey string) *Filestore {
-	return &Filestore{path: rootPath, caCertPath: caCert, privateKeyPath: privateKey}
+	return &Filestore{
+		path:           rootPath,
```

Comment:
Good question.  I think it was in case there was anything we wanted to change or initialize in the future if we moved to a different model.  I'm OK removing.